src/cljs/rems/user_settings.cljs

```diff
@@ -80,10 +80,32 @@
                    (when first-time? ; to avoid an infinite loop if the settings fail to save (e.g. unsupported language in cookies)
                      [[:check-if-should-save-language!]]))})))
 
+(rf/reg-event-fx
+ :flash-message-error
+ (fn [_ user-settings-error]
+   ;; TODO: refactor this part
```

Comment:
So is there still a TODO here? The if below looks a bit weird. Why not show the status code always or never?